jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/AutocompleteTests.java

```diff
@@ -0,0 +1,68 @@
+package io.github.epam.angular.tests.elements.complex;
+
+import io.github.epam.*;
+import org.testng.annotations.*;
+
+import static io.github.com.StaticSite.*;
+import static io.github.com.pages.AngularPage.*;
+import static io.github.epam.site.steps.States.*;
+import static org.testng.Assert.*;
+
+public class AutocompleteTests extends TestsInit {
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        angularPage.shouldBeOpened();
+    }
+
+    @Test
+    public void verifyAutoCompleteOverview() {
+        autocompleteSection.autocompleteOverview.setValue("F", "FloridaPopulation: 20.27M");
+        assertEquals(autocompleteSection.autocompleteOverview.getValue(), "Florida");
```

Comment:
Use [JDI asserts ](https://jdi-docs.github.io/jdi-light/?java#jdi-asserts]) only in basic tests

Reply:
Updated with JDI Asserts.